client/layout/masterbar/masterbar-cart/masterbar-cart-button.tsx

```diff
@@ -59,17 +86,21 @@ export function MasterbarCartButton( {
 	const onClose = () => setIsActive( false );
 	const tooltip = String( translate( 'My shopping cart' ) );
 
+	const cartCount = responseCart?.products?.length;
+	const icon = cartIcon || <CartIcon newItems={ !! responseCart.products } active={ isActive } />;
+
 	return (
 		<>
 			<MasterbarItem
 				className="masterbar-cart-button"
 				alwaysShowContent
-				icon={ <CartIcon newItems={ !! responseCart.products } active={ isActive } /> }
+				icon={ icon }
 				tooltip={ tooltip }
 				onClick={ onClick }
 				isActive={ isActive }
 				ref={ cartButtonRef }
 			/>
+			{ showCount && <MasterBarCartCount cartCount={ cartCount } /> }
```

Comment:
Can you add a screenshot of this to the PR description?

Reply:
Added a screenshot.